README.md

```diff
@@ -5,29 +5,23 @@
 
 `nix-installer` is an opinionated, **experimental** Nix installer.
 
-> Try it on a machine/VM you don't care about!
->
-> ```bash
-> curl -L https://install.determinate.systems/nix | sh -s -- install
-> ```
+
+```bash
+curl -L https://install.determinate.systems/nix | sh -s -- install
+```
 
 ## Status
 
 `nix-installer` is **pre-release and experimental**. It is not ready for high reliability use! *Please* don't use it on a business critical machine!
 
-Planned support:
+Current and planned support:
 
-* [x] Multi-user x86_64 Linux with systemd init, no SELinux
-* [x] Multi-user aarch64 Linux with systemd init, no SELinux
-* [x] Multi-user x86_64 MacOS
-    + Note: User deletion is currently unimplemented, you need to use a user with a secure token and `dscl . -delete /Users/_nixbuild*` where `*` is each user number.
-* [x] Multi-user aarch64 MacOS
+* [x] Multi-user Linux (aarch64 and x86_64) with systemd init, no SELinux
+* [x] Multi-user MacOS (aarch64 and x86_64)
     + Note: User deletion is currently unimplemented, you need to use a user with a secure token and `dscl . -delete /Users/_nixbuild*` where `*` is each user number.
 * [x] Valve Steam Deck
-* [ ] Multi-user x86_64 Linux with systemd init, with SELinux
-* [ ] Multi-user aarch64 Linux with systemd init, with SELinux
-* [ ] Single-user x86_64 Linux
-* [ ] Single-user aarch64 Linux
+* [ ] Multi-user Linux with systemd init & SELinux
```

Comment:
Since you qualify single-user below, should we also qualify this here?  "(aarch64 and x86_64)"